utils/check_clang_tidy_results.py

```diff
@@ -9,7 +9,41 @@
 
 # Checks list: https://clang.llvm.org/extra/clang-tidy/checks/list.html
 SUPPRESSED_CHECKS = {
-    '[abseil-string-find-startswith]',
+    # Checks we don't care about and want to suppress permanently
```

Comment:
From my point of view there is no agreement on which checks to enable and which to surpress. So until there might be an agreement I am unhappy about such a comment implying agreement.